### PR TITLE
Custom Domain for Microservice

### LINKED ISSUE #361: Custom Domain Name for OrcaBus (MetadataManager)
We should start configure API Gateway custom domain name so each microservice could start to make use of each other.

### PR DESCRIPTION
**Pull request by williamputraintan**
- Add custom domain name for microservice api-gateway
	- metadata-manager -> `metadata`
	- file-manager -> `file`
	- sequence-run-manager -> ~`sequence-run`~ `sequence`
	- workflow-manager -> `workflow` 
- api-gw CW logs limited for 2 years in prod, and 14 days in dev/stg

Resolve #361

### CODE REVIEW

File: lib/workload/stateless/stacks/sequence-run-manager/deploy/stack.ts [hunk 1]
```diff
@@ -114,6 +115,7 @@ export class SequenceRunManagerStack extends Stack {
     const srmApi = new ApiGatewayConstruct(this, 'ApiGateway', {
       region: this.region,
       apiName: 'SequenceRunManager',
+      customDomainNamePrefix: 'sequence-run',
```
Comment (victorskl) [inline lib/workload/stateless/stacks/sequence-run-manager/deploy/stack.ts:118]: In this case, could you change it to just `sequence` pls.

File: skel/django-api/deploy/stack.ts [hunk 2]
```diff
@@ -117,6 +118,7 @@ export class ProjectNameStack extends Stack {  // FIXME change construct name
     const srmApi = new ApiGatewayConstruct(this, 'ApiGateway', {
       region: this.region,
       apiName: 'SequenceRunManager',
+      customDomainNamePrefix:'sequence-run',
```
Comment (victorskl) [inline skel/django-api/deploy/stack.ts:121]: Yup. Just `sequence` as noun pattern.

**Review by victorskl — APPROVED** [review]
LGTM. Just minor request.

**Comment by williamputraintan** [comment]
Updated! Will merge once test passed!

**Pull request merged**